api/Headers.json

```diff
@@ -21,7 +21,7 @@
             "version_added": "39"
           },
           "firefox_android": {
-            "version_added": "44"
+            "version_added": "39"
```

Comment:
Is this based on testing Firefox for Android or mirroring? I wouldn't trust mirroring alone, maybe the data is different on purpose currently.